src/Components/Lists/EquipmentDetailList/EquipmentDetailList.js

```diff
@@ -0,0 +1,62 @@
+// Import Components
+import React from 'react';
+import PropTypes from 'prop-types';
+import EquipmentDetailCard from '../../Cards/EquipmentDetailCard/EquipmentDetailCard';
+
+// Import Stylings
+import './EquipmentDetailList.css';
+
+// Import Icons
```

Comment:
Missing import? @tranleduyan

Reply:
Addressed! I just put their as to sort out if in the future, we need to put icons in or not. But I think it can be misleading and not look good so i have removed this empty comments in everywhere.